### ISSUE TITLE
Fluid unification

### ISSUE THREAD

**Issue by uhloin** [AlmostReliable/almostunified#106]
### Describe the feature/addition!

How about unification of pair of fluids (they are not unified by default)
1. Honey and 2. Experience and may be 3. Latex
Used in mods: Cyclic, Industrial Foregoing, Bumblezone, Sophisticated Backpacks, Thermal Expansion, Mob Grinding Tools

### Why would like to see this feature/addition?

It's not good to grind 2 liquids with the same name

**Comment by rlnt** [comment]
https://github.com/AlmostReliable/almostunified/wiki/FAQ#are-liquids-unified

Besides that, NeoForge provides a new common fluid ingredient type but not a lot of mods have adapted to this system yet so it's not feasible to support this any time soon. We'll see what 1.22 brings when AU switches to NeoForge only and whether mods have adapted by then.

<blockquote><img src="https://opengraph.githubassets.com/90338c85ae2d98763387ff4137e7b461eaf9fad73637ae155f49efc9044962f2/AlmostReliable/almostunified" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/AlmostReliable/almostunified/wiki/FAQ">FAQ</a></strong></div><div>A Minecraft mod to unify resources. Contribute to AlmostReliable/almostunified development by creating an account on GitHub.</div></blockquote>

**Comment by uhloin** [comment]
Understood. Fortunately, there are very small amount of liquids which need unification - not more than 10. So may be some hardcode approach may work without common system?

**Comment by rlnt** [comment]
No, it's not that easy. It will not "just work" even though there might be a small amount of fluids.
